### PR TITLE
Refacto css

### PR DESCRIPTION
**Pull request by DavidGeismarLtd**

### CODE REVIEW

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <section class='text-center home-hero py-12 px-8'>
   <h1 class='heading1 text-black max-w-5xl m-auto'>Un catalogue incisif de formations décisives pour votre carrière.</h1>
-  <p class='text-black text-lg font-light m-auto max-w-md mt-8'>Masterclass, DPC, attestations universitaires, le meilleur pour votre parcours professionnel.</p>
```
[the remark this answers is not in the document]
Comment (DavidGeismarLtd) [reply app/views/static_pages/home.html.erb:3]: I will rebase the branch with master and fix the conflict if it emerges then

File: app/helpers/category_helper.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module CategoryHelper
+  def format(name)
```
[the remark this answers is not in the document]
Comment (DavidGeismarLtd) [reply app/helpers/category_helper.rb:2]: `format_category_name` WDYT ?

[the remark this answers is not in the document]
Comment (ManyBaby) [reply app/helpers/category_helper.rb:2]: > `format_category_name` WDYT ?

Yes perfect

Comment (ManyBaby) [inline app/helpers/category_helper.rb:2]:
```suggestion
  def format_category_name(name)
```

**Pull request merged**